cypress/support/step-definitions/spinner-steps.js

```diff
@@ -0,0 +1,7 @@
+import { spinner } from '../../locators/spinner';
+
+const SPINNER_CLASS_PREFIX = 'carbon-spinner--';
+
+Then('Spinner {word} is set to {string}', (word, value) => {
```

Comment:
how `word` parameter is used here ?

Reply:
for cucumber step to recognize `as` or `size`